<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapaMundi</class>
 <widget class="QMainWindow" name="MapaMundi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/continentes (1).png);
color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_oceania3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 149);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_asia6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_america">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>371</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 105, 30);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_europa1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>-20</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 89, 239);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_oceania">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 197, 149);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_asia4">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>240</y>
      <width>71</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fondo_africa">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>261</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(16, 255, 24);
border: none;
background-repeat: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ranking">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgorund-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_asia3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_europa">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 89, 239);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_africa1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(16, 255, 24);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_asia">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_asia1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_asia5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 250, 80);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>fondo_africa</zorder>
   <zorder>fondo_america</zorder>
   <zorder>fondo_oceania</zorder>
   <zorder>fondo_asia6</zorder>
   <zorder>fondo_asia4</zorder>
   <zorder>fondo_europa</zorder>
   <zorder>fondo_africa1</zorder>
   <zorder>fondo_oceania3</zorder>
   <zorder>fondo_europa1</zorder>
   <zorder>fondo_asia</zorder>
   <zorder>fondo_asia1</zorder>
   <zorder>fondo_asia3</zorder>
   <zorder>fondo_asia5</zorder>
   <zorder>label</zorder>
   <zorder>lbl_ranking</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
